scripts/test-truncating-of-client-hello.py

```diff
@@ -188,7 +188,7 @@ def run_all(conversations):
         try:
             runner.run()
         except:
-            # TBD: print(traceback.format_exc())
+            print(traceback.format_exc())
```

Comment:
just to be clear, the output (including tracebacks) can be printed after all tests were run, but it needs to be clearly labelled, and this traceback saved.